airgun/entities/subscription.py

```diff
@@ -176,6 +176,11 @@ def delete(self, entity_name):
     def read_subscriptions(self):
         """Return subscriptions table"""
         view = self.navigate_to(self, 'All')
+        wait_for(
```

Comment:
we are trying to read the table 2 times here. which is not a good way. Better way would be just add this line 

`view.wait_displayed()`

Reply:
ah yeah, this is definitely the better way to do this. fuzzy late night brain :) 

If that ends up not being specific enough, you can always use wait_displayed on a specific element, as well.

I.e. `view.table.wait_displayed`

Reply:
okay